scripts/restore

```diff
@@ -31,13 +31,15 @@ db_name=$(ynh_app_setting_get --app=$app --key=db_name)
 db_user=$db_name
 phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 
+fpm_footprint=$(ynh_app_setting_get --app=$app --key=fpm_footprint)
+fpm_usage=$(ynh_app_setting_get --app=$app --key=fpm_usage)
```

Comment:
Those variables may not be defined at restore time, right ? I mean if scripts/config was never called before backup.

Reply:
Ah i see the install helper creates those settings.